docs/src/components/ui/AlphaBanner.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import Link from "next/link";
+import { styled } from "linaria/react";
+import { media } from "../../themes";
+
+// We don't use the theme here as this is a quickfix, and it doesn't change
+// depending on dark/light mode.
+const alphaBannerBackground = "#4F00A3" as const;
+const alphaBannerText = "#F2F2F2" as const;
+const alphaBannerCTA = "#C4C4C4" as const;
+const alphaBannerBraket = "#EDCF00" as const;
+
+const AlphaBannerContainer = styled.section`
+  font-family: ChivoRegular, sans-serif;
+  user-select: none;
+  z-index: 100;
```

Comment:
This z-index interferes with the popup from `TOOLS` which shows on hover over the tools button:

![image](https://github.com/user-attachments/assets/640d664d-0f93-4646-91f6-b7468f8837b7)


```suggestion
  z-index: 5;
```